examples/nlp/language_modeling/conf/megatron_gpt_config.yaml

```diff
@@ -72,6 +72,9 @@ model:
   # use GPTModel from megatron.core
   mcore_gpt: True
 
+  # Skip training and do only validation step
+  skip_train: True
```

Comment:
Do you want to setting this to True by default?

Reply:
oh, just a typo. It should be False